Tools/FitFractions.hpp

```diff
@@ -25,6 +26,17 @@
 namespace ComPWA {
 namespace Tools {
 
+struct FitFraction {
+  FitFraction(std::string name, double val, double err)
+      : Name(name), Value(val), Error(err) {}
```

Comment:
We could use `ComPWA::FitParameter` here and avoid another class

Reply:
Hmm I don't like this kind of reusing, since it will lead to a bad design eventually. A fit fraction is not a fit parameter in my opinion. Its just some value with an error that is calculated from a fit result (and an intensity). If there are some problems or downside we have to think about where in our current design the trouble arises